src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -295,7 +299,17 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         // handoff from an already-started PTY process.
         if (!_inPipe)
         {
-            THROW_IF_FAILED(_CreatePseudoConsoleAndPipes(dimensions, PSEUDOCONSOLE_RESIZE_QUIRK | PSEUDOCONSOLE_WIN32_INPUT_MODE, &_inPipe, &_outPipe, &_hPC));
+            DWORD flags = PSEUDOCONSOLE_RESIZE_QUIRK | PSEUDOCONSOLE_WIN32_INPUT_MODE;
+
+            if constexpr (Feature_VtPassthroughMode::IsEnabled())
+            {
+                if (_passthroughMode)
+                {
+                    WI_SetFlag(flags, PSEUDOCONSOLE_PASSTHROUGH_MODE);
```

Comment:
This may not compile in Release. You are using a preprocessor define to control whether the constant is visible, but a `constexpr if` (which _must compile successfully_) wherein you are using the constant.

Reply:
Actually, you're gating the presence of the member using the preprocessor as well.

Try a build from the commandline with `/p:WindowsTerminalBranding=Release`

Reply:
I'd say, keep the member and the constant defined and only gate whether the code actually touches them based on `if constexpr`